codemods/inline-style-prop/index.ts

```diff
@@ -71,34 +81,83 @@ export default function transformer(file: FileInfo, api: API) {
         styleAttribute.value.expression.type === "Identifier"
       ) {
         const objectName = styleAttribute.value.expression.name;
+
+        // Check to see if the object is a variable in the module scope
+        // if not, we handle it below
+        let foundVariables = false;
+
         root.findVariableDeclarators(objectName).forEach((path) => {
+          foundVariables = true;
           if (path.value.init.type === "ObjectExpression") {
             const node = path.value.init;
 
             node.properties = convertObjectProperties(
+              file.path,
               j,
               node.properties,
               needsFlexRemapping(node.properties),
             );
           }
         });
 
+        if (!foundVariables) {
```

Comment:
if the variable isn't in scope, that means it is either imported, or a function argument. so we log to ensure the engineer verifies all instances are converted